views/index.erb

```diff
@@ -0,0 +1,24 @@
+<html>
+  <head>
+  <title>Kelly's Website</title>
+  </head>
+  <body>
+  <h1>Kelly's Website</h1>
```

Comment:
Everything from here below should be indented since it is nested within the `body` tag